### ISSUE TITLE
Sort order - ignore capitalization

### ISSUE THREAD

**Issue by dfleck** [Burning-Man-Earth/iBurn-iOS#81]
Looking at Art, some of the items are out of order (Sysimetsa, THIRST, THUNDERBIRDS, Tara mechanical, Tell Me Yours)(and Tree of Tenere, Tryst with Technology, the Cranebow, the Solace Gemstone) It seems that capitalization is throwing it off.
Also, maybe it would be nice to skip "The" so that The Bear would be under B instead of T
Also it would be nice to ignore accents to allow a search for Tenere to find Ténéré.

**Comment by chrisballinger** [comment]
Good catch, it's surprising that capitalization would affect sort order. 

As far as searching alternative symbols, ideally we would have a way to denormalize: https://github.com/yapstudios/YapDatabase/issues/106
